test/automation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-automation",
-  "version": "1.71.0",
```

Comment:
Was this change intentional? It looks like this version hasn't been bumped upstream in vscode: https://github.com/microsoft/vscode/blob/main/test/automation/package.json

Reply:
Nope, not sure where that came from. Thank you for catching it!